config/README.md

```diff
@@ -3,6 +3,33 @@
 This folder contains kustomizations for the Baremetal Operator. They have
 traditionally been used through the [deploy.sh](../tools/deploy.sh) script,
 which takes care of generating the necessary config for basic-auth and TLS.
+
```

Comment:
Not loving that we actually have two copies of this to maintain. Can we put this in one place only?

Reply:
Yes. But could we do that in another pr?

Reply:
You need to push rebase etc so why can't we have a readme fixed in this PR? It is a minor fix.

Reply:
Because the two README has been there before this, so IMO it doesn't make sense to consolidate them here. And we might need to discuss how to have common README for both of them as well.